homeassistant/components/life360/device_tracker.py

```diff
@@ -1,432 +1,267 @@
+                if place_id not in logged_places:
+                    logged_places.append(place_id)
+                    new_places.append(place)
+            if new_places:
+                msg = f"Places from {circle['name']}:"
+                for place in new_places:
+                    msg += f"\n- name: {place['name']}"
+                    msg += f"\n  latitude: {place['latitude']}"
+                    msg += f"\n  longitude: {place['longitude']}"
+                    msg += f"\n  radius: {place['radius']}"
+                LOGGER.debug(msg)
+
+        new_entities = []
+        for member_id, member in coordinator.data["members"].items():
+            tracked_by_account = tracked_members.get(member_id)
+            if new_member := not tracked_by_account:
+                tracked_members[member_id] = entry.unique_id
+                LOGGER.info("Member: %s", member[ATTR_NAME])
+            if (
+                new_member
+                or tracked_by_account == entry.unique_id
+                and not new_members_only
+            ):
+                new_entities.append(Life360DeviceTracker(coordinator, member_id))
+        if new_entities:
+            async_add_entities(new_entities)
+
+    process_data(new_members_only=False)
+    account["unsub"] = coordinator.async_add_listener(process_data)
+
+
+class Life360DeviceTracker(CoordinatorEntity, TrackerEntity):
+    """Life360 Device Tracker."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, member_id: str) -> None:
+        """Initialize Life360 Entity."""
+        super().__init__(coordinator)
+        self._attr_attribution = ATTRIBUTION
+        self._attr_unique_id = member_id
+
+        self._data = coordinator.data["members"][self.unique_id]
+
+        self._attr_name = self._data[ATTR_NAME]
+        self._attr_entity_picture = self._data[ATTR_ENTITY_PICTURE]
+
+        self._prev_data = self._filtered_data()
+
+    def _filtered_data(self) -> dict[str, Any]:
+        # Filter out data that should be ignored when checking if anything has changed
+        # since last update. For now that's address, since that often constantly changes
+        # (back and forth between two values), even when nothing else changes. If it
+        # isn't filtered out, there will be way more state changes which would basically
+        # be useless (and spam the database, and possibly the log.)
+        return {k: v for k, v in self._data.items() if k != ATTR_ADDRESS}
+
+    @property
+    def _options(self) -> Mapping[str, Any]:
+        """Shortcut to config entry options."""
+        return cast(Mapping[str, Any], self.coordinator.config_entry.options)
+
+    @callback
+    def add_to_platform_start(
+        self,
+        hass: HomeAssistant,
+        platform: EntityPlatform,
+        parallel_updates: asyncio.Semaphore | None,
+    ) -> None:
+        """Start adding an entity to a platform."""
+        platform.entity_namespace = self._options.get(CONF_PREFIX)
```

Comment:
Why do we do this instead of just setting name on the entity? Entity namespace is legacy.

Reply:
Ok, I'll get rid of the prefix/namespace.